swatch-metrics/deploy/clowdapp.yaml

```diff
@@ -285,6 +287,8 @@ objects:
                   secretKeyRef:
                     name: ${INVENTORY_SECRET_KEY_NAME}
                     key: ${INVENTORY_SECRET_KEY_NAME_PREFIX}password
+              - name: EVENT_SOURCE
+                value: ${EVENT_SOURCE}
```

Comment:
Let's set this value directly to `prometheus` instead of parameterizing it in lines 96-97.  When the swatch-metrics-rhel clowdapp becomes part of this file, we'll set it directly as "rhelemeter" for that one.

Reply:
+1 Done